doc/features/tuning-policies/README.md

```diff
@@ -143,20 +143,24 @@ A retry policy centralizes the handling of query retries, minimizing the need fo
 
 The retry policy interface consists of four methods:
 
-- `#onReadTimeout(requestInfo, consistency, received, blockFor, isDataPresent)`: determines what to do when the driver
+- `#onReadTimeout(info, consistency, received, blockFor, isDataPresent)`: determines what to do when the driver
 gets a `ReadTimeoutException` response from a Cassandra node.
-- `#onUnavailable(requestInfo, consistency, required, alive)`: determines what to do when the driver gets an 
+- `#onUnavailable(info, consistency, required, alive)`: determines what to do when the driver gets an 
 `UnavailableException` response from a Cassandra node.
-- `#onWriteTimeout(requestInfo, consistency, received, blockFor, writeType)`: determines what to do when the driver gets
+- `#onWriteTimeout(info, consistency, received, blockFor, writeType)`: determines what to do when the driver gets
 a `WriteTimeoutException` response from a Cassandra node
-- `#onRequestError(requestInfo, consistency, err)`: defines whether to retry and at which consistency level on an 
+- `#onRequestError(info, consistency, err)`: defines whether to retry and at which consistency level on an 
 unexpected error, invoked in the following situations:
     - On a client timeout, while waiting for the server response , being the error an instance of 
     `OperationTimedOutError`.
     - On a connection error (socket closed, etc.).
     - When the contacted host replies with an error, such as `overloaded`, `isBootstrapping`, `serverError`, etc. In 
     this case, the error is instance of `ResponseError`
 
-A default and base retry policy is included.
+A default and base retry policy is included, along with `IdempotenceAwareRetryPolicy` that considers.
```

Comment:
I think since we are doing a minor release, we can jump to what the java driver ultimately did, which is, if the query is non-idempotent, don't even pass it to the retry policy on a WriteTimeout / RequestError and log that this is happening the first time we encounter a WriteTimeout/Request Error for non idempotent operation:

https://github.com/datastax/java-driver/blob/3.1.2/driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java#L375-L380

https://github.com/datastax/java-driver/blob/3.1.2/driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java#L530-L540